File: src/phase2/linkedlist/doublylinkedlist/LruCachePlayground.java
package phase2.linkedlist.doublylinkedlist;

import java.util.Arrays;
import java.util.List;

public class LruCachePlayground {
    public static void main(String[] args) {
        //10, 10, 20, 30, 20, 20, 50, 60, 70, 30, 90, 90
        List<Integer> list = Arrays.asList(10, 20, 30, 20, 20, 50, 60, 70, 30, 90, 90);
        LruCache lruCache = new LruCache(10, 5);
        list.forEach(lruCache::add);
    }
}

File: src/phase2/linkedlist/grind75/LruCachePlayground.kt
package phase2.linkedlist.grind75

fun main() {

    var lruCache = LRUCache(2)

    lruCache.put(2,1)
    lruCache.put(3,2)

    println(lruCache.get(3))
    println(lruCache.get(2))
    lruCache.put(4,3)

    println(lruCache.get(2)) //check if 3 is removed

    println(lruCache.get(3)) //check this

    println(lruCache.get(4))

}